force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -48,12 +48,17 @@ export default class DonationHistoryTable extends LightningElement {
     
     // eslint-disable-next-line @lwc/lwc/no-async-await
     async connectedCallback() {
-        getDonationHistory({contactId: this.contactId})
+        getTotalNumberOfRecords({contactId: this.contactId})
+        .then(data => {
+            if(data){
+                this.totalNumberOfRows = data[0].expr0;
```

Comment:
I think this has to be managed by the service in the backend

Reply:
I left a bit of feedback above on this, but I think that `getDonationHistory()` should return as much of the data needed for the initial load of the app as possible, and the total number of records is a good candidate for including in that response.

Reply:
done